apps/api/v2/src/modules/conferencing/repositories/conferencing.respository.ts

```diff
@@ -0,0 +1,25 @@
+import { PrismaReadService } from "@/modules/prisma/prisma-read.service";
+import { PrismaWriteService } from "@/modules/prisma/prisma-write.service";
+import { Injectable } from "@nestjs/common";
+
+import { GOOGLE_MEET_TYPE } from "@calcom/platform-constants";
+
+@Injectable()
+export class ConferencingRepository {
+  constructor(private readonly dbRead: PrismaReadService, private readonly dbWrite: PrismaWriteService) {}
+
+  async findConferencingApps(userId: number) {
+    return this.dbRead.prisma.credential.findMany({
+      where: {
+        userId,
+        type: { endsWith: "_video" },
+      },
+    });
+  }
+
+  async findGoogleMeet(userId: number) {
```

Comment:
NIT: we can also use `getByTypeAndUserId` from the credentials repository, it does the exact same thing,